build_info/logcleaner.control

```diff
@@ -0,0 +1,10 @@
+Package: logcleaner
+Version: @DEB_LOGCLEANER_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
```

Comment:
Give yourself some credit in the Author field below this line, even though it's not "Debian-friendly" (the field doesn't exist)